efcms-service/serverless.yml

```diff
@@ -381,6 +381,26 @@ resources:
             MetricsEnabled: true
 
 functions:
+  createDocument:
+    handler: src/documents/createDocumentLambda.createDocument
```

Comment:
petitioner (user) should be able to start a case so the auth error there might be caused by something environmental? I just checked on our stg environment and it works.

Reply:
The browser records this error:

```
Uncaught UnauthorizedError: Unauthorized
    at Object._callee$ (http://localhost:1234/index.dev.1ac7f1b6.js:100481:19)
    at tryCatch (http://localhost:1234/index.dev.1ac7f1b6.js:5844:40)
    at Generator.invoke [as _invoke] (http://localhost:1234/index.dev.1ac7f1b6.js:6070:22)
    at Generator.prototype.(anonymous function) [as next] (http://localhost:1234/index.dev.1ac7f1b6.js:5896:21)
    at asyncGeneratorStep (http://localhost:1234/index.dev.1ac7f1b6.js:100443:103)
    at _next (http://localhost:1234/index.dev.1ac7f1b6.js:100445:194)
    at http://localhost:1234/index.dev.1ac7f1b6.js:100445:364
    at new Promise (<anonymous>)
    at Object.<anonymous> (http://localhost:1234/index.dev.1ac7f1b6.js:100445:97)
    at Object.getCaseTypes (http://localhost:1234/index.dev.1ac7f1b6.js:100495:18)
```

Because it never hits the backend, nothing is logged in to stderr or stdout.

Reply:
I built this on another system, and I can't log in as `petitioner` at all.

Which is the correct behavior: being able to log in as `petitioner`, or not?

Reply:
I built this on _three_ systems in total, and the inability to log in was my error (I think I was building off a sprint 5 commit) — in fact, I get precisely the original error in every case, in which `petitioner` has the ability to log in, but no permission to actually do anything.